packages/webapp/src/containers/Map/saga.js

```diff
@@ -74,7 +82,45 @@ export function* setSpotlightToShownSaga({ payload: spotlights }) {
   }
 }
 
+export const bulkUploadSensorsInfoFile = createAction(`bulkUploadSensorsInfoFileSaga`);
+
+export function* bulkUploadSensorsInfoFileSaga({ payload: { file } }) {
+  try {
+    yield put(bulkSensorsUploadLoading());
+    const { farm_id } = yield select(userFarmSelector);
+    const formData = new FormData();
+    formData.append('sensors', file);
+    const fileUploadResponse = yield call(axios.post, bulkUploadSensorsInfoUrl(), formData, {
+      headers: {
+        'Content-Type': 'multipart/form-data',
+        Authorization: 'Bearer ' + localStorage.getItem('id_token'),
+        farm_id: farm_id,
+      },
+    });
+
+    if (fileUploadResponse.status === 200) {
```

Comment:
@Cali0707 I think you and @anant00729 should coordinate about how you want to handle errors. Since right now its showing an error on the frontend even when sensors are being added to the database.

Reply:
@fsayyid I think this may have been resolved by the commit I just pushed - would you be able to test this and confirm?

Reply:
I think for the most part this is fine now. It's not likely that users will be claiming sensors owned by others or non existing sensors. Here's what we can do for now:

@Cali0707 Send a 400 error in the case of occupied or non existing sensor.  
@anant00729 on the frontend if there are any successful sensors added, we can enqueue the success snackbar. If there is a 400 also enqueue as an error "Non existing or occupied sensors could not be registered". Or something like that, so users see both snackbars.